examples/scripts/its_rollout_example.py

```
@@ -0,0 +1,231 @@
+        max_concurrency=args.max_concurrency,
+    )
+
+    rollouts_per_iter = args.prompt_batch_size * args.group_size
+    candidates_per_iter = rollouts_per_iter * args.budget
+
+    print("ITS Hub + GRPO Training")
+    print("=" * 60)
+    print(f"Model:              {args.model_path}")
+    print(f"Data:               {args.data_path} ({len(tasks)} tasks)")
+    print(f"Algorithm:          {args.algorithm} (budget={args.budget})")
+    print(f"Backend:            art (single GPU)")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)